udev/Makefile

```diff
@@ -24,20 +24,25 @@ install-reload:
 
 check:
 	@for RULE in *-hdmi2usb*.rules; do \
-		echo -n "Checking $$RULE.."; \
+		echo -n "Checking installed $$RULE.."; \
 		[ -e /etc/udev/rules.d/$$RULE ] || exit 1; \
 		diff -u $$RULE /etc/udev/rules.d/$$RULE || exit 1; \
 		echo " Good!"; \
 	done
 	@for HELP in hdmi2usb-*-helper.sh; do \
-		echo -n "Checking $$HELP.."; \
+		echo -n "Checking installed $$HELP.."; \
 		[ -e /etc/udev/rules.d/$$HELP ] || exit 1; \
 		diff -u $$HELP /etc/udev/rules.d/$$HELP || exit 1; \
 		echo " Good!"; \
 	done
-	if ! id | grep -qF '(video)'; then \
+	@if ! id | grep -qF '(video)'; then \
 		echo "Not a member of the video group"; exit 1; \
 	fi
+	@for HELP in hdmi2usb-*-helper.sh; do \
```

Comment:
This should probably be in a `test:` target... `check:` is more for "checking things are installed correctly and up to date".